README.md

```diff
@@ -0,0 +1,103 @@
+## Generate a new Maven project ##
+
+- Run below command from Command line tool
+
+```text
+mvn archetype:generate -DartifactId=selenium-learning -DgroupId=com.mytest -DarchetypeGroupId=org.apache.maven.archetypes -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4
```

Comment:
<div>
<h3>Code Review Agent Run <a href = https://github.com/Amruta101998/selenium-automation/pull/2/files#2122405823>#64877</a> - 05/21/2024, 04:52 pm</h3>

<p>🔴 High importance</p>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The Maven command for generating a project is highly customizable and might lead to performance issues if not properly configured. The specified archetypeVersion might be outdated, leading to the use of older libraries which could have performance implications.
</div>
<div id="fix">
<b>Fix:</b> Update the Maven archetype version to the latest to ensure the use of optimized and up-to-date libraries. This can lead to better performance and access to newer features.
</div>
<details>
<summary>
Code suggestion
</summary>
<div>
<pre><code>
 -mvn archetype:generate -DartifactId=selenium-learning -DgroupId=com.mytest -DarchetypeGroupId=org.apache.maven.archetypes -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4
 +mvn archetype:generate -DartifactId=selenium-learning -DgroupId=com.mytest -DarchetypeGroupId=org.apache.maven.archetypes -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.7
</code></pre>
</div>
</details>
<h4><a href="https://github.com/Amruta101998/selenium-automation/pull/2/files#2122405823"><-- Code Review Agent Run#64877</a></h4>
<i>AI Code Review powered by <img src="https://bito.ai/wp-content/uploads/2023/10/Logo-Bito-Black-cropped.svg" alt="Bito Logo" width="50" height="25"></i>
</div>
<br>

<br>

</div>